src/Model/Behavior/ShadowTranslateBehavior.php

```diff
@@ -378,6 +378,9 @@ public function groupTranslations($results)
     {
         return $results->map(function ($row) {
             $translations = (array)$row['_i18n'];
+            if (empty($translations) && $row->get('_translations')) {
```

Comment:
After a recent discussion about potentially hidden bugs using empty, it is usually encouraged to use sth else.
E.g.

    if (!$translations && ...)

or

 if (count($translations) === 0 && ...)